src/Application.vala

```diff
@@ -54,6 +54,8 @@ public enum ENotes.Key {
 namespace ENotes {
     public ENotes.Services.Settings settings;
     public ENotes.Window window;
+    public ClockSettings clockSettings;
```

Comment:
Two comments :)

- In vala, variable names use are `named_like_this`
- Change to private since clock_settings is not in use in any other place

For more about code style, check this: https://elementary.io/docs/code/reference#variable-names-class-names-function-names

Reply:
Just saw that the variable names were like this from the time indicator.... our bad... We've learned a lot from the past few years! :sweat_smile: